### PR TITLE
Cody for VS Code v0.8 release blog post

### PR DESCRIPTION
**Pull request by toolmantim**

### CODE REVIEW

**Review by kalanchan — APPROVED** [review]
WOOOH, looks good! Thanks for pulling to together and changing up the tone of it 😄

File: content/blogposts/2023/cody-vscode-0-8-release.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: "Cody for VS Code v0.8 release"
+authors:
+  - name: Kalan
+    url: https://handbook.sourcegraph.com/team/#kalan-chan
+  - name: Tim
+    url: https://handbook.sourcegraph.com/team/#tim-lucas
+publishDate: 2023-08-20T10:00-07:00
```
Comment (kalanchan) [inline content/blogposts/2023/cody-vscode-0-8-release.md:8]: date change here woops

**Review by philipp-spiess — DISMISSED** [review]

**Review by philipp-spiess — APPROVED** [review]
LGTM

File: content/blogposts/2023/cody-vscode-0-8-release.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+We’ve just released [Cody for VS Code](https://marketplace.visualstudio.com/items?itemName=sourcegraph.cody-ai) v0.8, with faster autocomplete, improved recipes (now commands), new chat layout, improved code smell detection, custom commands, and more.
+
+Here’s what’s new in v0.8, and a few of the improvements we’ve made since the [June release](https://about.sourcegraph.com/blog/cody-in-sourcegraph-5-1).
+
+### Improved autocomplete
+
+This release builds on a number of autocomplete improvements we’ve made since June:
+
+* Reduced p75 response time by 40% to &lt;1300ms, with autocomplete now using Claude Instant 1.2
```
Comment (philipp-spiess) [inline content/blogposts/2023/cody-vscode-0-8-release.md:25]:
```suggestion
* Reduced single-line p75 response time by 40% to &lt;1300ms, with autocomplete now using Claude Instant 1.2
```

I wonder if we should be concrete here, not to confuse people?

**Pull request merged**